models/shipping-method/src/shipping-method/shipping-method-draft/presets/sample-data-goodstore/usa.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  KeyReference,
+  LocalizedString,
+} from '@commercetools-test-data/commons';
+import { TaxCategoryDraft } from '@commercetools-test-data/tax-category';
+import type { TTaxCategoryDraft } from '@commercetools-test-data/tax-category';
+import * as ZoneRateDraft from '../../../../zone-rate/zone-rate-draft';
+import type { TShippingMethodDraftBuilder } from '../../../types';
+import * as ShippingMethodDraft from '../../index';
+
+const standardTaxCategoryDraft = TaxCategoryDraft.presets.sampleDataGoodStore
+  .standardTaxCategory()
+  .build<TTaxCategoryDraft>();
+
+const usaShippingMethod = (): TShippingMethodDraftBuilder =>
+  ShippingMethodDraft.presets
+    .empty()
+    .key('us-delivery')
+    .name('US Delivery')
+    .localizedDescription(
+      LocalizedString.presets
+        .empty()
+        ['en-US']('US Shipping')
+        ['en-GB']('US Shipping')
+        ['de-DE']('US Shipping')
```

Comment:
Should we make other shipping methods' descriptions match this? Just a nit.

Reply:
This is pure placeholder 😄